apps/admin/integration-testing/e2e/support/auth.ts

```diff
@@ -50,10 +50,11 @@ export function mockElectionManagerCardInsertion({
  * Enters the PIN into the PIN pad.
  */
 export async function enterPin(page: Page): Promise<void> {
-  await page.waitForSelector('text=Enter the card PIN to unlock.');
+  await page.getByText('Enter the card PIN to unlock.').waitFor();
```

Comment:
To be more consistent with our RTL testing, I want to move away from the `text=something` style selectors. Instead, get the "locater" and make assertions against it.

In this case, Playwright docs recommend against using `waitForSelector` and instead recommend using `waitFor` with a locator.